api/CSSPageRule.json

```diff
@@ -123,6 +123,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "CSSPageDescriptors": {
```

Comment:
Really wasn't sure what to call this subfeature to mean "derives from this now"

Reply:
Not sure what you mean by "derives from". This is a property (not an interface), so I guess the property's value changed?

Reply:
Yes, sorry. The type of the property changed to CSSPageDescriptors

Reply:
```suggestion
        "returns_CSSPageDescriptors": {
```

Reply:
```suggestion
        "type_CSSPageDescriptors": {
```